### DISCUSSION TITLE
slog device fail & no power loss

### DISCUSSION THREAD

**Discussion by ghost** [openzfs/zfs#13214] [Q&A] [answered]
Is there no problem at all if your slog device fails while there is no power loss around that time?

Because the same data is still in memory and not discarded after writing to slog, but only after the write is 100% complete?
Is that how zfs works?

**Comment by rincebrain** [comment, marked as answer]
Yes, the data from the slog is never read in normal operation, so if it falls over and there's not a sudden crash or power loss before things get written, it's perfectly fine.
